internal/pkg/state/registry/registry.go

```diff
@@ -235,20 +233,6 @@ func (s *Registry) ConfigRowsFrom(config ConfigKey) (rows []*ConfigRowState) {
 	return rows
 }
 
-// func (s *Registry) SetIgnoredConfigsOrRows(ctx context.Context, fs filesystem.Fs, path string) error {
```

Comment:
Why is this commented out?

Reply:
[2c14c77](https://github.com/keboola/keboola-as-code/pull/2136/commits/2c14c7783abac54d60b6f12e32664d6b2704cd74)

Reply:
it's removed